src/nspawn/nspawn.c

```diff
@@ -3289,6 +3291,11 @@ static int outer_child(
 
         log_debug("Outer child is initializing.");
 
+        r = load_os_release_pairs_container_prefix_env("/", &os_release_pairs);
```

Comment:
maybe call the function "load_os_release_pairs_with_prefix()" and then specify the prefix as additional argument here.